src/js/helpers/covid19/budgetCategoriesHelper.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * agencyHelper.js
+ * Created by James Lee 6/10/20
+ */
+
+import { apiRequest } from "../apiRequest";
+
+export const fetchDisasterSpending = (type, params) => apiRequest({
+    url: `v2/disaster/${type}/spending/`,
+    method: 'post',
+    data: params,
+    isMocked: true
+});
+
+export const fetchDisasterSpendingCount = (type, params) => apiRequest({
+    url: `v2/disaster/${type}/count/`,
+    method: 'post',
+    data: params,
+    isMocked: true
+});
+
+export const fetchDefCodes = () => apiRequest({
```

Comment:
Can be removed after Jon's PR is merged - we also may consolidate all the COVID api helpers into a single file, but this is fine for now!

Reply:
Done!